apps/platform/src/app/components/project/ProjectCard.tsx

```diff
@@ -2,59 +2,49 @@ import Image from 'next/image';
 import Link from 'next/link';
 
 import AddToCart from '@/app/components/cart/AddToCart';
+import { getProjectProgressBGColor } from '@/app/helpers/projectHelper';
 
 interface ProjectCardProps {
 	className?: string;
+	progressPercentage: number;
 }
 
-export default function ProjectCard({ className }: ProjectCardProps) {
+export default function ProjectCard({
+	className,
+	progressPercentage,
```

Comment:
nitpick: to have it more concise when typing out component:
```suggestion
	progress,
```